.github/workflows/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         run: mix test
       - name: Run Credo
         run: mix credo --strict
-      - name: Run Excoveralls
-        run: mix coveralls.github
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # - name: Run Excoveralls
+      #   run: mix coveralls.github
+      #   env:
+      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why is this commented out?

Reply:
The coveralls service has a monthly runs limit since we're under active development this limit will be quickly reached and the CI's workflow will fail.

Reply:
As additional info: we're using a matrix strategy between OTP and the elixir versions making the excoveralls service run 4 times per/workflow.